pset-functions.js

```diff
@@ -90,6 +109,28 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 2:')
 
 // Add your code below this line
+function wordCount(phrase) {
+  // Have to split the sentence into words
+  // first get rid of possible commas --
+  const nocomma=phrase.replace(',',"")
+  const words=nocomma.split(" ")
+  const nWord={}
+  //loop through the number of words in the sentence and
+  //check for each of the words we have already identifies
+  words.forEach((word) => {
+    //    console.log(nWord[word])
+    if (nWord[word] === undefined) {
+    nWord[word] = 1
+    } else {
+    nWord[word]++
+    }
+  })  // closing the loop
+  return nWord
+} // closing the function
+
+const phrase="Baby shark, doo doo doo doo doo doo"
+console.log("Test:" + phrase)
+console.log(wordCount(phrase))
```

Comment:
@AtleLohrmann Nice job her creating this function here to take to create the object with the `phrase` and value of each time a word is repeated.  Good job on the splitting the string and checking each word to either set to one if it doesn't exist else increment the value by 1.
Take a look at this solution [here](https://github.com/jsr20190826/hw-02-functions-solution/blob/a4d789161886ff115657133c279062df8a4ca72d/pset-functions.js#L103)